docs/zz_generated.kubermaticConfiguration.ce.yaml

```diff
@@ -47,6 +47,22 @@ spec:
       requests:
         cpu: 200m
         memory: 150Mi
+  # Applications contains configuration for Application settings.
+  applications:
+    # DefaultApplicationCatalog contains configuration for the default application catalog.
+    defaultApplicationCatalog:
+      # Applications is a list of application definition names that should be installed in the master cluster.
+      # If not set, all the applications from the catalog are installed.
+      applications: null
+      # Enable is used to enable the installation of application definitions in the master cluster.
+      enable: false
+    # SystemApplications contains configuration for system applications.
+    systemApplications:
+      # Applications is a list of application definition names that should be installed in the master cluster.
+      # If not set, all the applications from the catalog are installed.
+      applications: null
+      # Disable is used to disable the installation of system application definitions in the master cluster.
+      disable: false
```

Comment:
Note for reviewer:

The enable/disable flags are opposite on purpose since the default app catalog is EE only and optional, while system apps need to be installed by default.